vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+
+- name: create minio group
+  become: yes
+  group:
+    name: minio
+    gid: 9002
+    state: present
+- name: create minio user
+  become: yes
+  user:
+    name: minio
+    group: minio
+    shell: /sbin/nologin
```

Comment:
I see in other places we use shell: /bin/false . What is the difference?

Reply:
it seems with /sbin/nologin the shell prints a message "this account is not available" but with /bin/false the shell just terminates with no message.  Not much difference but we should use /sbin/nologin everywhere.  https://unix.stackexchange.com/questions/10852/whats-the-difference-between-sbin-nologin-and-bin-false